csp/wsgi.py

```diff
@@ -13,6 +13,7 @@
 from dj_static import Cling
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "csp.settings")
-
+import django
+django.setup()
```

Comment:
this is not needed, you only need that if you want to use your models in a python console, please do not update wsgi files and settings